src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -215,6 +215,13 @@ namespace DataHeatBalance {
     extern int const MixingSourceZonesOnly;
     extern int const AllZones;
 
+    // Parameter for zone air flow mass balancing method
+    extern int const AdjustMixingOnly;
+    extern int const AdjustReturnOnly;
+    extern int const AdjustMixingThenReturn;
+    extern int const AdjustReturnThenMixing;
+    extern int const NoAdjustReturnAndMixing;
```

Comment:
Hmmmm I wish this had been an enumeration rather than more global integers hanging out.  Oh well.

Reply:
I will change to enumeration along with other follow up changes.